installer/charts/rhtap-integrations/templates/tests/test.yaml

```diff
@@ -0,0 +1,64 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    helm.sh/hook: test
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+  labels:
+    {{- include "common.labels" . | nindent 4 }}
+  name: {{ printf "%s-test" .Chart.Name }}
+spec:
+  restartPolicy: Never
+  serviceAccountName: {{ .Release.Name }}
+  initContainers:
+    #
+    # Copying the scripts that will be used on the subsequent containers, the
+    # scripts are shared via the "/scripts" volume.
+    #
+    {{- include "common.copyScripts" . | nindent 4 }}
+  containers:
+    #
+    # Test the SCM tokens.
+    #
+    {{- $scm := "github" }}
+    {{- $scmSecretName := (printf "rhtap-%s-integration" $scm) }}
+    {{- $scmSecretObj := (lookup "v1" "Secret" .Release.Namespace $scmSecretName) }}
+    {{- if $scmSecretObj }}
+    - name: test-{{ $scm }}-token
+      image: quay.io/codeready-toolchain/oc-client-base:latest
```

Comment:
Can we use an image from either registry.redhat.io or registry.access.redhat.com?

This will run in the user's infrastructure.

Reply:
Done